packages/account/src/Components/additional-kyc-info-modal/form-config.tsx

```diff
@@ -8,11 +8,11 @@ export type TListItem = {
     /**
      * The text of the item (e.g. 'United Kingdom', 'Germany', etc.)
      */
-    text: string;
+    text?: string;
```

Comment:
Why are both text and value optional?

Reply:
![Screenshot 2024-01-02 at 5 17 07 PM](https://github.com/binary-com/deriv-app/assets/103026762/5dbf8399-8686-492a-989d-856edd519fbd)
If I remove optional value, typescript will complain in personal-details-form.tsx

Reply:
Cant we do it in the component level instead of doing it here like Partial or DeepPartial or something 🤔